packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -40,172 +50,185 @@ export default function create_manifest_data({
 	fallback = `${get_runtime_path(config)}/components`,
 	cwd = process.cwd()
 }) {
-	/**
-	 * @param {string} file_name
-	 * @param {string} dir
-	 */
-	function find_layout(file_name, dir) {
-		const files = config.extensions.map((ext) => posixify(path.join(dir, `${file_name}${ext}`)));
-		return files.find((file) => fs.existsSync(path.resolve(cwd, file)));
-	}
-
-	/** @type {string[]} */
-	const components = [];
-
 	/** @type {import('types').RouteData[]} */
 	const routes = [];
 
-	const default_layout = posixify(path.relative(cwd, `${fallback}/layout.svelte`));
-	const default_error = posixify(path.relative(cwd, `${fallback}/error.svelte`));
-
-	/**
-	 * @param {string} dir
-	 * @param {string[]} parent_id
-	 * @param {Array<string|undefined>} layout_stack // accumulated __layout.svelte components
-	 * @param {Array<string|undefined>} error_stack // accumulated __error.svelte components
-	 */
-	function walk(dir, parent_id, layout_stack, error_stack) {
-		/** @type {Item[]} */
-		const items = [];
-
-		fs.readdirSync(dir).forEach((basename) => {
-			const resolved = path.join(dir, basename);
-			const file = posixify(path.relative(cwd, resolved));
-			const is_dir = fs.statSync(resolved).isDirectory();
+	/** @type {Map<string, Unit>} */
+	const units = new Map();
 
-			const ext = is_dir
-				? ''
-				: config.extensions.find((ext) => basename.endsWith(ext)) ||
-				  config.kit.endpointExtensions.find((ext) => basename.endsWith(ext));
+	/** @type {Tree} */
+	const tree = new Map();
 
-			if (ext === undefined) return;
-
-			const name = basename.slice(0, basename.length - ext.length);
+	// set default root layout/error
+	tree.set('', {
+		error: posixify(path.relative(cwd, `${fallback}/error.svelte`)),
+		layouts: {
+			default: { file: posixify(path.relative(cwd, `${fallback}/layout.svelte`)), name: 'default' }
+		}
+	});
 
-			if (name.startsWith('__') && !specials.has(name)) {
-				throw new Error(`Files and directories prefixed with __ are reserved (saw ${file})`);
-			}
+	const routes_base = path.relative(cwd, config.kit.files.routes);
```

Comment:
Unit tests pass with this on Windows:
```suggestion
	const routes_base = posixify(path.relative(cwd, config.kit.files.routes));
```